app/javascript/article-form/components/Help.jsx

```diff
@@ -235,6 +235,13 @@ export class Help extends Component {
             </a>{' '}
             to add rich content such as Tweets, YouTube videos, etc.
           </li>
+          <li>
```

Comment:
Not sure about the wording for this. Any improvements welcome.

Reply:
"In addition to images for the post's content, you can also drag and drop a cover image"

Maybe something short like this?